databases/week2/1hyf_lesson2_class_exercise.sql

```diff
@@ -0,0 +1,38 @@
+-- Get all the tasks assigned to Pavel;
+SELECT
+  task.title,
+  user.name
+FROM
+  (
+    task
+    join user_task ON task.id = user_task.task_id
+  )
+  join user ON user.id = user_task.user_id
+WHERE
+  user.name = "Pavel Brushneen";
+--Find how many tasks each user is responsible for;
```

Comment:
the comment and -- suggested to have space in between, like the 1st comment you wrote:
"-- Get"